tests/test_server.py

```diff
@@ -864,9 +864,33 @@ async def test_get_resource_actions(postgresql_client, client, clienthelper, ser
     )
     assert result.code == 200
 
+    #  adding a resource action with its change field set to "created" to test the get_resource_actions
+    #  filtering on resources with changes
+    rvid_r1_v1 = "std::File[agent1,path=/etc/file1],v=1"
+    await data.Resource.new(
+        environment=uuid.UUID(environment),
+        status=const.ResourceState.available,
+        resource_version_id=rvid_r1_v1,
+        attributes={"purge_on_delete": False, "purged": True, "requires": []},
+    ).insert()
+
+    later_action_id = uuid.uuid4()
+    start_time = datetime.now()
+    resource_action = data.ResourceAction(
+        environment=uuid.UUID(environment),
+        version=1,
+        resource_version_ids=[rvid_r1_v1],
+        action_id=later_action_id,
+        action=const.ResourceAction.deploy,
+        started=start_time,
+        change=const.Change.created,
+    )
+    await resource_action.insert()
```

Comment:
Why directly query the database here instead of using the API. The API is a lot more stable over time than the database schema. A small change in the schema of the ResourceAction table might break this test case.